### PR TITLE
Test

### PR DESCRIPTION
**Pull request by tblenahan**
renaming and removing server logging function

### CODE REVIEW

**Comment by KyleNeubarth** [comment]
Hi this is a test comment on our first pull request of the semester

**Review by kolin-newby — APPROVED** [review]
Minor changes, all look good.

File: src/logger/helper.go [hunk 1]
```diff
@@ -8,20 +8,20 @@ import (
 )
```
Comment (KyleNeubarth) [inline src/logger/helper.go:9]: Looks all good here, just a test comment

**Review by KyleNeubarth — APPROVED** [review]
10/10 quality code

**Pull request merged**